extensions/wrapper/wrapper-broker-api/src/main/java/de/sovity/edc/ext/wrapper/api/broker/BrokerServerResource.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.broker;
+
+import de.sovity.edc.ext.wrapper.api.broker.model.CatalogPageQuery;
+import de.sovity.edc.ext.wrapper.api.broker.model.CatalogPageResult;
+import de.sovity.edc.ext.wrapper.api.broker.model.ConnectorPageQuery;
+import de.sovity.edc.ext.wrapper.api.broker.model.ConnectorPageResult;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+/**
+ * Broker Server API Endpoints to be included in our generated EDC API Wrapper Clients
+ */
+@Path("wrapper/broker")
```

Comment:
We might reuse the broker API for connector in the future. The broker-feature thus, may become a catalog-caching feature or an enhanced-catalog. Please align API paths accordingly to be future-proof